src/tests/ftest/pool/management_race.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python3
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+import random
+from apricot import TestWithServers
+from thread_manager import ThreadManager
+from command_utils_base import CommandFailure
+
+class PoolManagementRace(TestWithServers):
+    """
+    Epic: Create system level tests that cover pool management race boundary tests and
+          functionality.
+    Testcase:
+          DAOS-8463: Pool management integrity with race boundary conditions
+    Test Class Description:
+          Start DAOS servers, create pools and delete the pool while query and list.
+    :avocado: recursive
+    """
+
+    def del_recreate_query_and_list_pools(
+            self, del_recreate=True, pool_num=1, thread_num=1, test_loop=2):
```

Comment:
It would be cleaner to pass in the actual `TestPool` object here instead of the index into `self.pool`:
```suggestion
            self, del_recreate=True, this_pool, thread_num=1, test_loop=2):
```